XCUITest/OnboardingTest.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  OnboardingTest.swift
+//  XCUITest
+//
+//  Created by Sawyer Blatz on 5/30/18.
+//  Copyright © 2018 Mozilla. All rights reserved.
+//
+
+import XCTest
+
+class OnboardingTest: BaseTestCase {
+
+    override func setUp() {
+        super.setUp()
+    }
+    
+    override func tearDown() {
+        app.terminate()
+        super.tearDown()
+    }
+    
+    func testPressingDots() {
+        let stackElement = XCUIApplication().children(matching: .window).element(boundBy: 0).children(matching: .other).element.children(matching: .other).element.children(matching: .other).element(boundBy: 2)
```

Comment:
Is there a more reliable and concise way to reference the element, perhaps adding an identifier?